src/bindgen/index.md

```diff
@@ -12,26 +12,30 @@ them (so the compiler knows what symbols to use). Enter [bindgen][bindgen].
 
 Bindgen does all the hard work of parsing C-style header files and generating 
 the equivalent Rust function definitions. They've also got a 
-[great tutorial][tut] for getting started. We'll try to focus on the more high 
-level stuff which comes with using a C library from Rust, seeing as the zero 
-cost abstractions and high level way of thingking is probably why you're trying 
-use Rust in a C/C++ codebase in the first place!
+[great tutorial][tut] for getting started. This section focuses mostly on the
+high level stuff required when using a C library from Rust, seeing as the zero
+cost abstractions and high level way of thinking is probably why you're trying
+to use Rust in a C/C++ codebase in the first place!
 
 
 ## Getting Set Up
 
-We'll be building on top of the [great tutorial][tut] linked earlier to write
+> **Note:** `bzip2` must be installed on your system for this section.  
+> **For Windows**, The process for getting bzip2 header files
+> recognized is **unknown**. If you can help, please file an [issue][issue].
+
+You'll build off the [great tutorial][tut] linked earlier to write
```

Comment:
`we'll` is still in the first person.
`you'll` sounds demanding.  `You will do X.`
I like your sentence, so building off that.
How about taking the reader out of writing the code? (Since they're probably doing that anyway)
`continuing on from where the [original bindgen tutorial][tut] left off, an idiomatic Rust wrapper will be written around bzip2.`
I'm conflicted on whether to use `will be` or `is`.
`will be` sounds like there's a chance it won't happen.  While `is` makes sure it will happen, but sounds like the wrapper is already written before the code is seen.